packages/composites/proof-of-reserves/src/utils/addressValidator.ts

```diff
@@ -22,19 +22,18 @@ export const getValidAddresses = (
 ): AdapterResponse => {
   const validatedInput = { ...protocolOutput }
   if (!util.parseBool(validator.validated.data.disableAddressValidation)) {
-    validatedInput.result = validateAddresses(
+    validatedInput.data.result = validateAddresses(
       validator.validated.id,
       validator.validated.data.indexer,
-      validatedInput.result as any,
+      validatedInput.data.result as any,
     ) as any
   }
   if (!util.parseBool(validator.validated.data.disableDuplicateAddressFiltering)) {
-    validatedInput.result = filterDuplicates(
+    validatedInput.data.result = filterDuplicates(
       validator.validated.id,
-      validatedInput.result as any,
+      validatedInput.data.result as any,
```

Comment:
We actually use `data.result` to set `result` in `Requester.success()`. So both fields would be set to the same thing. Since all of the existing address list adapters are on v2 using this method, they'd all have it set properly